causal_testing/specification/capabilities.py

```diff
@@ -0,0 +1,75 @@
+"""
+This module contains the Capability and TreatmentSequence classes to implement
+treatment sequences that operate over time.
+"""
+from causal_testing.specification.variable import Variable
+
+
+class Capability:
+    """
+    Data class to encapsulate temporal interventions.
+    """
+
+    def __init__(self, variable: Variable, value: any, start_time: int, end_time: float):
```

Comment:
Should start_time be `int` and end_time `float`?

Reply:
Good spot. Fixed.